src/modules/profile/infos/phone.tsx

```diff
@@ -40,13 +44,30 @@ export const ProfileInfosPhone = observer(() => {
 
       <TextInput
         label="Phone"
-        value={profileFlowStore.phone}
-        onChangeText={(phone) => (profileFlowStore.phone = phone)}
+        value={phone}
+        onChangeText={phone => SetPhone(phone)}
         keyboardType="number-pad"
       />
 
       <View style={styles.buttonContainer}>
-        <Button onPress={navigation.goBack}>Done</Button>
+        <Button
+          onPress={async () => {
+            if (userStore.profile._id) {
+              await userStore.patchProfile(userStore.profile._id, {
```

Comment:
I don't see why we await we the end of the result. Let's do it in "optimistic update" mode